### DISCUSSION TITLE
Downloading Error

### DISCUSSION THREAD

**Discussion by TimeKeyRoll** [ConsenSys-archive/truffle#5342] [General Q&A]
Hi guys! I have downloaded `ganache` on `window7`. its succesfully downloaded. however when i run it it shows blank (white screen). i have tried to redownload as well. but there is same error. is there anyone who has experienced this? could anyone please help?

**Comment by apdan510** [comment]
@MasterofBlockchain Thanks for reaching out. Please uninstall that instance of Ganache and try this installer: https://github.com/trufflesuite/ganache-ui/releases/download/v2.5.4/Ganache-2.5.4-win-setup.exe

Kind regards,
Alex

**Reply by TimeKeyRoll** [reply]
But nothing like this mentioned on their website?

**Reply by apdan510** [reply]
@MasterofBlockchain the website link for Ganache is an "appx" file formally formatted for Windows 10/11 since Microsoft fully deprecated Windows 7 Support a few years back. Let me know if the installer works.

**Reply by TimeKeyRoll** [reply]
I have downloaded as you provided the link

But there is same error: white screen.

**Reply by apdan510** [reply]
@MasterofBlockchain Can you share any more details about what happens? Does it go through installation prompts? Or when you open the .exe it just shows a blank white screen?

**Reply by TimeKeyRoll** [reply]
that is all what i see everytime.

![ganache](https://user-images.githubusercontent.com/101450342/180278461-e0060f42-953a-4ac8-aa98-f7a2d562f935.PNG)

**Reply by TimeKeyRoll** [reply]
No i am able to download it. its gets downloaded smoothly. but when it comes to open the the Ganache after launch then there you go with the white screen which is mentioned above.

**Comment by apdan510** [comment]
@MasterofBlockchain 

There could possibly be some compatibility issues here. Please uninstall Gananche then restart your PC completely and try this installer: https://github.com/trufflesuite/ganache-ui/releases/download/v1.2.2/ganache-setup-1.2.2.exe

Worse case scenario we can use the non user interface version Ganache package

Let me know if it works. 

Kind regards,
Alex | Truffle Support

**Comment by TimeKeyRoll** [comment]
well, after the download i cant even install it. there is this error.. please check.

![3](https://user-images.githubusercontent.com/101450342/180298785-69ffa4a6-6060-4b5a-b6c3-dd5a038b1e48.PNG)

**Reply by apdan510** [reply]
@MasterofBlockchain Unfortunately these are definitely compatibility issues. I believe we can bypass the ganache user interface and use the ganache npm package that runs solely in a terminal let me know if that is something you would like to do.

Thanks!
